### PR TITLE
www/nginx: lodash update

### PR DESCRIPTION
**Pull request by fabianfrz**
* dependency update

### CODE REVIEW

File: www/nginx/Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 PLUGIN_NAME=		nginx
 PLUGIN_VERSION=		1.14
+PLUGIN_REVISION=        1
```
Comment (fabianfrz) [inline www/nginx/Makefile]: this is not yet fixed, may be 1.15 if there will be features as well.

**Comment by AdSchellevis** [comment]
@fabianfrz the diff for lodash in frr doesn't appear to be different (looks like there only an extra end of line in it)

**Comment by fabianfrz** [comment]
@AdSchellevis I copy and pasted the lib from https://lodash.com/ and maybe it was already 4.17.11.

**Comment by AdSchellevis** [comment]
@fabianfrz probably, better leave frr unchanged then.

**Comment by AdSchellevis** [comment]
@fabianfrz thanks!

**Comment by fabianfrz** [comment]
@AdSchellevis that went fast…

**Comment by fichtner** [comment]
GitHub is still complaining: Upgrade lodash to version 4.17.13 or later.

**Comment by fabianfrz** [comment]
we can switch to https://www.npmjs.com/package/lodash .14 which has been released 5 days ago.

**Comment by fichtner** [comment]
yes please :)

**Pull request merged**